parsons/zoom/zoom.py

```diff
@@ -275,3 +275,149 @@ def get_webinar_registrants(self, webinar_id):
         tbl = self._get_request(f"webinars/{webinar_id}/registrants", "registrants")
         logger.info(f"Retrieved {tbl.num_rows} webinar registrants.")
         return tbl
+
+    def get_meeting_poll(self, meeting_id, poll_id) -> Table:
+        """
+        Get a specific poll for a given meeting ID
+
+        `Args`:
+            meeting_id: int
+                Unique identifier for Zoom meeting
+            poll_id: int
+                Unique identifier for poll
+
+        `Returns`:
+            Parsons Table of all polling responses
+        """
+
+        endpoint = f"meetings/{meeting_id}/polls/{poll_id}"
+        tbl = self._get_request(endpoint=endpoint, data_key="questions")
+
+        if type(tbl) == dict:
+            logger.debug(f"No poll data returned for poll ID {poll_id}")
+            return Table(tbl)
+
+        logger.info(
+            f"Retrieved {tbl.num_rows} rows of metadata [meeting={meeting_id} poll={poll_id}]"
+        )
+
+        return tbl
+
+    def get_meeting_all_polls(self, meeting_id) -> Table:
+        """
+        Get all polls for a given meeting ID
+
+        `Args`:
+            meeting_id: int
+                Unique identifier for Zoom meeting
+
+        `Returns`:
+            Parsons Table of all polling responses
+        """
+
+        endpoint = f"meetings/{meeting_id}/polls"
+        tbl = self._get_request(endpoint=endpoint, data_key="polls")
+
+        if type(tbl) == dict:
+            logger.debug(f"No poll data returned for meeting ID {meeting_id}")
+            return Table(tbl)
+
+        logger.info(f"Retrieved {tbl.num_rows} polls for meeting ID {meeting_id}")
+
+        return tbl
+
+    def get_past_meeting_poll(self, meeting_id) -> Table:
+        """
+        List poll results of a past meeting.
+
+        `Args`:
+            meeting_id: str
```

Comment:
Here's a meeting_id type that's still string

Reply:
good catch! I updated a few from existing functions too for consistency